dlpy/utils.py

```diff
@@ -500,7 +501,17 @@ def caslibify_context(conn, path, task='save'):
             remaining_path += sep
 
         if caslib is not None:
-            yield caslib, remaining_path
+            access_subdir = conn.retrieve('caslibinfo', _messagelevel = 'error',
```

Comment:
not sure what you are trying to do here?

Reply:
Get the permission of the subdirectory.